.github/workflows/dynamic-readme-example.yaml

```diff
@@ -0,0 +1,17 @@
+name: update-templates
```

Comment:
Added this as an example for folks to copy and paste on the new repo being created.